src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+/**
+ * The Ui class encapsulates interactions with
+ * the user
+ */
+
+public class Ui {
+	private static String logo = " ____        _        \n"
+			+ "|  _ \\ _   _| | _____ \n"
+			+ "| | | | | | | |/ / _ \\\n"
+			+ "| |_| | |_| |   <  __/\n"
+			+ "|____/ \\__,_|_|\\_\\___|\n";
+
+	/**
+	 * Prints the welcome message whenever
+	 * the program is started
+	 */
+	public static void showWelcomeMessage() {
+		String welcomeMsg = "Hello from\n" + logo +
+				"\n" + "Hello! I'm Duke\nWhat can I do for you?";
+		System.out.println(welcomeMsg);
+	}
+
+	/**
+	 * Prints the goodbye message whenever
+	 * the program is terminated.
+	 */
+	public static void showGoodbyeMessage() {
+		System.out.println("Bye. Hope to see you again soon!");
+	}
```

Comment:
Great function names! You explain the named entity to readers accurately and at a sufficient level of detail.